app/global-translations/locale-en.json

```diff
@@ -3510,6 +3510,8 @@
     "label.anchor.passwordPreferences":"Password Preferences",
     "label.view.passwordPreferences":"Define standards for enforcing the usage of stronger passwords",
     "label.heading.passwordPreferences":"Password Preferences",
+    "label.simple" : "Password must be at least 1 character and not more that 50 characters long",
+    "label.secure" : "Password must be at least 6 characters, no more than 50 characters long, must include at least one upper case letter, one lower case letter, one numeric digit and no space",
```

Comment:
while this is perfectly fine, it would have been nicer to create a more distinct label (say label.password.preferences.simple etc) which makes it more obvious as to where the label is used